File: CurrencyConverterApp/app/src/main/java/com/ramonsilva/currencyconverterapp/MainActivity.java
package com.ramonsilva.currencyconverterapp;

import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.util.Log;
import android.view.View;
import android.widget.EditText;
import android.widget.RadioButton;
import android.widget.TextView;
import android.widget.Toast;

public class MainActivity extends AppCompatActivity {

    public void convertCurrency(View view) {
        EditText currencyEditTextView = findViewById(R.id.idEditTextCurrency);
        if(!currencyEditTextView.getText().toString().isEmpty()) {
            double currency = Double.parseDouble(currencyEditTextView.getText().toString());
            double coeficiente = 0.0;
            String simbol = "R$";

            RadioButton rdoBtnDolar = findViewById(R.id.idRdoBtnDolar);
            RadioButton rdoBtnReal = findViewById(R.id.idRdoBtnReal);

            if (rdoBtnDolar.isChecked()) {
                simbol = "U$";
                coeficiente = 3.9;
            } else {
                coeficiente = (1 / (3.9));
            }
            double result = (currency * coeficiente);
            String strResult = Double.toString(result);
            Log.i("result", strResult);
            Toast.makeText(MainActivity.this, simbol+" "+strResult, Toast.LENGTH_LONG).show();
            currencyEditTextView.setText("");
        }
    }

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
    }
}

File: CurrencyAppKotlin/app/src/main/java/com/ramonsilva/currencyappkotlin/MainActivity.kt
package com.ramonsilva.currencyappkotlin

import android.support.v7.app.AppCompatActivity
import android.os.Bundle
import android.provider.MediaStore
import android.widget.EditText
import android.widget.RadioButton
import android.widget.Toast
import kotlinx.android.synthetic.main.activity_main.*

class MainActivity : AppCompatActivity() {


    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        idBtnConvert.setOnClickListener{
            convertCurrency();
        }
    }

    fun convertCurrency() {
        val editTextVal = findViewById<EditText>(R.id.idEditTextCurrency).text.toString();
        if(!editTextVal.isNullOrEmpty()) {
            val radioBtnRealToDolar = findViewById<RadioButton>(R.id.idRdoBtnRealToDolar);
            var coeficient = 2.5;
            var currency = "U$";
            if (radioBtnRealToDolar.isChecked) {
                currency = "R$";
                coeficient = 0.25;
            }
            val numberOfEditText = editTextVal.toDouble();
            val result = numberOfEditText * coeficient;

            val ajustedResult = String.format("%.2f", result);

            Toast.makeText(this@MainActivity, currency + ajustedResult, Toast.LENGTH_LONG).show();
            findViewById<EditText>(R.id.idEditTextCurrency).setText("");
        }
    }
}
